packages/vscode-extension/README.md

```diff
@@ -1,64 +1,93 @@
-SuiteCloud CLI for Node.js is another open-source SuiteCloud SDK tool. This tool has more available commands that you can use in combination with the ones available in SuiteCloud extension for Visual Studio Code.
-
-To work with both tools at the same time, do the following:
-1. Install SuiteCloud CLI for Node.js by following the instructions [here](../node-cli/README.md).
-2. Open a terminal inside of your SDF project, and enter:
-   ```javascript
-   suitecloud -h
-   ```
-3. You can now use SuiteCloud CLI for Node.js in your SDF projects while working in SuiteCloud extension for Visual Studio Code.
-
-![SuiteCloud CLI for Node.js combined with SuiteCloud extension for Visual Studio Code](resources/cliForNodejsDemo.gif)
-
-4. Set up a new account, or select an account that is already configured to use for your project.
-
-![SuiteCloud CLI for Node.js combined with SuiteCloud extension for Visual Studio Code](resources/setupAccountDemo.gif)
-
-## [Contributing](/CONTRIBUTING.md)
-SuiteCloud Extension for Visual Studio Code is an open source project. Pull Requests are currently not being accepted. See [CONTRIBUTING](/CONTRIBUTING.md) for details.
-
-## [License](/LICENSE.txt)
-Copyright (c) 2021 Oracle and/or its affiliates The Universal Permissive License (UPL), Version 1.0.
\ No newline at end of file
+- VS Code version 1.57 or higher
+
+Read the detailed list of prerequisites in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installation Requirements for SuiteCloud Extension for Visual Studio Code</a>.
+
+## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual Studio Code </font> </span>
+To install SuiteCloud Extension for Visual Studio Code, follow these steps:
+
+1. Launch Visual Studio Code and select the **Extensions** icon in the Activity Bar.
+2. In the search field, enter **SuiteCloud Extension for Visual Studio Code**.
+3. Click **Install**.
+4. Restart Visual Studio Code after installation.
+
+See more details about installation in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installing and Setting Up SuiteCloud Extension for Visual Studio Code</a>.
+
+## <font size="6"> <span style="color:gray"> Getting Started </font> </span>
+
+### <font size="5"> <span style="color:gray"> Active File in the Editor </font>
+In order for the extension to detect the working SDF project and run the commands against it, you first need to select a file from the current workspace and make it active in the editor.
+
+### <font size="5"> <span style="color:gray"> How to Trigger the Command Palette </font>
+Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. To trigger it, click **View** > **Command Palette** or use the relevant shortcut:
+* Ctrl+Shift+P if you are using Windows.
```

Comment:
I read the comment in Slack, I think you could add it here "or Linux OS".

Reply:
Done, @lfpvillegas, thanks for noticing it!